src/SCamera.cpp

```diff
@@ -0,0 +1,23 @@
+#include <QCameraInfo>
+#include <QCameraFocus>
+#include <QCameraViewfinderSettings>
+
+#include "SCamera.h"
+
+SCamera::SCamera()
+{
+//    setCamera(QCameraInfo::defaultCamera());
+    this->load();
+    QCameraViewfinderSettings viewfinderSettings;
+    viewfinderSettings.setResolution(640, 480);
+    viewfinderSettings.setMinimumFrameRate(15.0);
+    viewfinderSettings.setMaximumFrameRate(30.0);
+    this->setViewfinderSettings(viewfinderSettings);
+
+    QCameraFocus *cameraFocus = this->focus();
+    cameraFocus->setFocusMode(QCameraFocus::ContinuousFocus);
+    cameraFocus->setFocusPointMode(QCameraFocus::FocusPointAuto);
+
+    this->captureMode().setFlag(QCamera::CaptureVideo);
+    this->start();
```

Comment:
I think that all of this stuff can be set in Qml. I think that we don't need to create classes that are not part of our solution. Smb can have his own Camera type (doesn't want to use ours) and he needs to see in an example how to set his camera.
Also, another custom type makes our library to look a bit more complicated. We want it to keep user-friendly and easy to use.